modules/imgproc/test/ocl/test_warp.cpp

```diff
@@ -172,7 +172,7 @@ OCL_TEST_P(WarpAffine, Mat)
 {
     for (int j = 0; j < test_loop_times; j++)
     {
-        double eps = depth < CV_32F ? 0.04 : 0.06;
```

Comment:
I propose to convert it to conditions with explicit types. We have fp16, int64, bool that comes after fp64. Also it's hard to understand the condition.